generator/internal/scaffold_generator.cc

```diff
@@ -379,10 +386,20 @@ include(GoogleapisConfig)
 set(DOXYGEN_PROJECT_NAME "$title$ C++ Client")
 set(DOXYGEN_PROJECT_BRIEF "A C++ Client Library for the $title$")
 set(DOXYGEN_PROJECT_NUMBER "$${PROJECT_VERSION}$doxygen_version_suffix$")
-set(DOXYGEN_EXCLUDE_SYMBOLS "internal" "$library$_internal" "$library$_testing"
-                            "examples")
-set(DOXYGEN_EXAMPLE_PATH $${CMAKE_CURRENT_SOURCE_DIR}/samples
-                         $${CMAKE_CURRENT_SOURCE_DIR}/quickstart)
+set(DOXYGEN_EXCLUDE_SYMBOLS "internal")
+set(DOXYGEN_EXAMPLE_PATH $${CMAKE_CURRENT_SOURCE_DIR}/quickstart)
+
+unset(mocks_globs)
```

Comment:
I see you applied the changes to `vmwareengine/BUILD.bazel`, I am guessing as a test? Why not to vmwareengine/CMakeLists.txt`?

Reply:
> Why not to `vmwareengine/CMakeLists.txt`?

`vmwareengine/CMakeLists.txt` matches the scaffold, as is.

> I am guessing as a test?

I updated `vmwareengine/BUILD.bazel` for the sake of consistency. Of course, I forgot about `speech/BUILD.bazel`: https://github.com/googleapis/google-cloud-cpp/blob/main/google/cloud/speech/BUILD.bazel#L24-L29